openstack/blockstorage/extensions/schedulerhints/requests.go

```diff
@@ -0,0 +1,116 @@
+package schedulerhints
+
+import (
+	"regexp"
+
+	"github.com/gophercloud/gophercloud"
+)
+
+// SchedulerHints represents a set of scheduling hints that are passed to the
+// OpenStack scheduler.
+type SchedulerHints struct {
```

Comment:
It looks like these fields are defined in this schema:

https://github.com/openstack/cinder/blob/b8167a5c3e5952cc52ff8844804b7a5ab36459c8/cinder/api/schemas/scheduler_hints.py

The schema defines a `query` field of type `string. Let's add that here for completeness.

Reply:
OK